### PR TITLE
400 3303 pages preview url for home

### PR DESCRIPTION
**Pull request by bjorvack**
## Type

- Non critical bugfix

## Resolves the following issues

#3303

## Pull request description

Fixes the preview url on for the home page

### CODE REVIEW

File: src/Backend/Core/Js/Components/Meta.js [hunk 1]
```diff
@@ -88,6 +88,10 @@ export class Meta {
               url = data.data
               $(options.urlSelector).val(url)
               $(options.generatedUrlSelector).html(url)
+
+              if (url.trim() === 'home') {
+                $(options.generatedUrlSelector).val('')
```
Comment (carakas) [inline src/Backend/Core/Js/Components/Meta.js:93]: Shouldn't you just change the value of URL tot empty string before the lines 89 and 90?

Comment (carakas) [reply src/Backend/Core/Js/Components/Meta.js:93]: Also, wouldn't this one give issues if I want to name a different page home?
You should do this based on the page id

**Review by carakas — CHANGES_REQUESTED** [review]

File: src/Backend/Core/Js/Components/Meta.js (hunk 1 above)
Comment (bjorvack) [reply src/Backend/Core/Js/Components/Meta.js]: fixed d060df6dc12c31f525250398e1d0d541a030a923

**Comment by bjorvack** [comment]
@carakas refactored the Meta.js 4146cd3085d8db8e30ea2912ae498a8f76c1e960

**Review by carakas — APPROVED** [review]

**Comment by carakas** [comment]
Thank you @bjorvack

File: src/Backend/Core/Layout/Templates/FormLayout.html.twig [hunk 2]
```diff
@@ -370,9 +370,15 @@
   {% if form.parent.meta is defined %}
     {% set meta = form.parent.meta %}
     {% set slug = meta.children.url.vars.value %}
-    <a target="_blank" rel="noopener noreferrer" class="site-url" href="{{ meta.vars.detail_url }}/{{ slug }}">
-      {{ SITE_URL }}{{ meta.vars.detail_url }}/<span id="generatedUrl">{{ slug }}</span>
-    </a>
+    {% if slug == 'home' %}
+      <a target="_blank" rel="noopener noreferrer" class="site-url" href="{{ meta.vars.detail_url }}/">
```
Comment (Katrienvh) [inline src/Backend/Core/Layout/Templates/FormLayout.html.twig:374]: `/` is not needed in href?

Comment (bjorvack) [reply src/Backend/Core/Layout/Templates/FormLayout.html.twig]: @Katrienvh fixed c098435854e65e3209130c0a70c661fd6f58250b

**Review by Katrienvh — APPROVED** [review]

**Pull request merged**